tests/expected/function-contract/history/simple_fail.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Zfunction-contracts
+
+#[kani::ensures(|result| old(*ptr) == *ptr)]
```

Comment:
We can replace `|result|` with simply `||` if we don't even have a return value. Maybe it makes the case cleaner.